ui/src/wallet.ts

```diff
@@ -65,6 +65,16 @@ function getProvider(
         "ropsten",
         "66fa0f92a54e4d8c9483ffdc6840d77b"
       );
+    case ethereum.Environment.Rinkeby:
+      return new ethers.providers.InfuraProvider(
+        "rinkeby",
+        "de5e2a8780c04964950e73b696d1bfb1"
+      );
+    case ethereum.Environment.Mainnet:
+      return new ethers.providers.InfuraProvider(
+        "mainnet",
+        "7a19a4bf0af84fcc86ffb693a257fad4"
```

Comment:
Could you add a comment saying that you have access to this infura token?

Reply:
798a7a9b8f4f1f8ae4cf5acb5ef31ee8cef16959

Reply:
Could you add the same comment to the Rinkeby token :innocent:?

Reply:
I can't find anymore who's account it was 😬